docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -1,638 +1,772 @@
-_visualView = new VisualView();
-GradientVisual gradientVisualMap1 = new GradientVisual();
-
-/// Enter the visual view properties
-_visualView.ParentOrigin = ParentOrigin.TopLeft;
-_visualView.PivotPoint = PivotPoint.TopLeft;
-_visualView.Size2D = new Size2D(window.Size.Width, window.Size.Height);
-
-/// Create the gradient visual
-
-/// Add the gradient visual
-_visualView.AddVisual("gradientVisual1", gradientVisualMap1);
-```
-
-<a name="visualmap"></a>
-## Using Visual Maps
+# Visuals
 
-You can both create visuals and position, and then resize them within a control, using classes inherited from the [Tizen.NUI.VisualMap](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.VisualMap.html) class, which is a base class for visuals. The class encapsulates various visual properties, such as the size, offset, depth index, shader, mix color, and opacity. It also contains the transform map for the visual and provides a custom `Shader` property.
+Visuals are the main building blocks for UI components. They provide reusable rendering logic that is controlled by using properties and can be used by all components. They also respond to view size and color changes and can perform clipping at the renderer level.
 
-To use visual maps follow the steps:
+The properties specific to each individual visual are encapsulated in the corresponding class, such as a `Border` visual is defined in a `BorderVisual` class, a `Color` visual in a `ColorVisual` class, and so on. The properties that are common for all visual types are inherited from the [Tizen.NUI.VisualMap](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.VisualMap.html) class. To render a visual, it has to be added to a control. A container class [Tizen.NUI.BaseComponents.VisualView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.VisualView.html) controls any visual that you have added.
 
--   You can create a property map for a visual using a `Tizen.NUI.VisualMap`-inherited class.
+## Create visual
 
-    The following example illustrats part of the [Tizen.NUI.ColorVisual](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.ColorVisual.html) class, which defines a color visual property map:
+To create a visual using `BorderVisual` class, follow these steps:
 
+1.  Create a visual object of a desirable type:
+    ```csharp
+    BorderVisual _borderVisual = new BorderVisual();
     ```
-    public class ColorVisual : VisualMap
-
-    private Color _mixColorForColorVisual = null;
-
-    public Color Color
-    {
-        get
-        {
-            return _mixColorForColorVisual;
-        }
-        set
-        {
-            _mixColorForColorVisual = value;
-            UpdateVisual();
-        }
-    }
-    ```
-
--   You can create a visual from an output visual map:
 
+2.  Specify the properties required for the visual type. The following section provides detailed information about the properties that are mandatory for a given visual:
```

Comment:
I simply removed the second sentence. The idea was to inform the reader, that in the later part of the guide it will be specified which properties are obligatory are which are not.

Reply:
@mgrudzinska If so, related with the description in next step, how about adding "mandatory" to this line to specify  that there are mandatory/optional properties?

2.  Specify the mandatory properties required for the visual type. 
...
3. You can specify the optional properties of a given visual, ...
...

Reply:
I thought that the fact, that they are mandatory is clear, since I wrote that they are 'required', but of course I can add it.
now there will be no doubts, that they are mandatory
thanks